main.py

```
@@ -0,0 +1,245 @@
+import asyncio
+import time
+import uuid
+from typing import Dict
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)